operator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM REPLACE_BASE_IMAGE
+FROM lgarciaac/helm-app-operator:v0.0.1
```

Comment:
Is this change OK?  `lgarciaac` seems too specific. :)

Reply:
Sure, the right domain for this is _apicurito_ :), but i cant push there :(

Reply:
@EricWittmann any chances to have those two images under apicurito? 

from `lgarciaac/helm-app-operator:v0.0.1` to `apicurito/helm-app-operator:v0.0.1`
and `lgarciaac/apicurito-operator` to `apicurito/apicurito-operator`

Reply:
I don't have an `apicurito` domain in docker hub but I do have `apicurio` - so how about we rename the images to:

* apicurio/apicurito-helm-app-operator
* apicurio/apicurito-operator

And I can happily push them to docker hub.

Reply:
oh right I mean `apicurio` not `apicurito`. Please do so, retag and push the images to:
```
apicurio/apicurito-helm-app-operator
apicurio/apicurito-operator
```
And let me know so I test all over again and make appropriate changes in the code.

What about granting me the powers to do all this myself?? ;)

Reply:
Yeah that's probably better - what is your docker hub id?  Same as your GH id?

Reply:
nope, it is that, `lgarciaac`

Reply:
Oh wow, I'm dumb.  :)  Of course it's `lgarciaac`!  lol

Reply:
OK I've granted you access to the Apicurio docker hub - go nuts!

Reply:
@lgarciaaco ^^^^^

Reply:
Awesome!! Thx!